zebra-state/src/service/finalized_state/disk_db.rs

```diff
@@ -207,13 +208,12 @@ impl ReadDisk for DiskDb {
     where
         C: rocksdb::AsColumnFamilyRef,
     {
-        // Empty column families return invalid forward iterators.
-        //
-        // Checking iterator validity does not seem to cause database hangs.
-        !self
+        // Check if the requested column family is empty
+        let column_family = self
             .db
             .iterator_cf(cf, rocksdb::IteratorMode::Start)
-            .valid()
```

Comment:
`valid()` and `status()` are not available anymore in rocksdb 0.19. I think what i did here is maybe not the most efficient way to do it but it seems to work.

Reply:
They have moved to the `DBRawIteratorWithThreadMode` type, I'll push a PR shortly.